src/Microsoft.TemplateEngine.Abstractions/Installer/InstallResult.cs

```diff
@@ -10,32 +11,41 @@ namespace Microsoft.TemplateEngine.Abstractions.Installer
     /// </summary>
     public sealed class InstallResult : InstallerOperationResult
     {
-        private InstallResult(InstallRequest request, IManagedTemplatePackage templatePackage)
+        private InstallResult(InstallRequest request, IManagedTemplatePackage templatePackage, Dictionary<int, IList<string>>? vulnerabilities = null)
             : base(templatePackage)
         {
             InstallRequest = request;
+            Vulnerabilities = vulnerabilities;
         }
 
-        private InstallResult(InstallRequest request, InstallerErrorCode error, string errorMessage)
+        private InstallResult(InstallRequest request, InstallerErrorCode error, string errorMessage, Dictionary<int, IList<string>>? vulnerabilities = null)
              : base(error, errorMessage)
         {
             InstallRequest = request;
+            Vulnerabilities = vulnerabilities;
         }
 
         /// <summary>
         /// <see cref="InstallRequest"/> processed by <see cref="IInstaller.InstallAsync"/> operation.
         /// </summary>
         public InstallRequest InstallRequest { get; private set; }
 
+        /// <summary>
+        /// Vulnerabilities from installed package.
```

Comment:
```suggestion
        /// Gets vulnerabilities from installed package.
```

please update also other occurrences of documentation. It would be good to explain what is the key and value or use list named tuples instead of dictionary.